web/app_options.js

```diff
@@ -492,7 +498,9 @@ class AppOptions {
   }
 
   static get(name) {
-    return userOptions[name] ?? defaultOptions[name]?.value ?? undefined;
+    return userOptions.has(name)
```

Comment:
Could we avoid `null` or `undefined` values in options and then just avoid `has` & `get` to just use `get` ?

Reply:
Unfortunately I don't believe that possible in general, hence these changes, given:
 - The values that may be used with browser-options.
 - Upcoming changes in PR #18465.